actiontext/app/models/action_text/rich_text.rb

```diff
@@ -36,7 +36,16 @@ class RichText < Record
     serialize :body, coder: ActionText::Content
     delegate :to_s, :nil?, to: :body
 
+    ##
+    # :method: record
+    #
+    # Returns the associated record.
     belongs_to :record, polymorphic: true, touch: true
+
+    ##
+    # :method: embeds
+    #
+    # Returns the <tt>ActiveStorage::Blob</tt>s of the embedded files.
```

Comment:
Action Text documentation is now in markdown. `<tt>` and `+` are not necessary anymore.